static/js/zamboni/files.js

```diff
@@ -112,6 +112,32 @@ var Highlighter = {
         // highlighter will try to replace the original element in the
         // DOM when it's done.
 
+        // Verify that we actually support the used brush
+        // see https://github.com/mozilla/addons-server/issues/4552 for more
+        // details. This shows an alert if a brush is not supported.
+        var discoveredBrushes = SyntaxHighlighter.brushes;
+
+        if (discoveredBrushes) {
+            var brushes = [];
+
+            for (var discoveredBrush in discoveredBrushes) {
+                var aliases = discoveredBrushes[discoveredBrush].aliases;
+
+                if (aliases == null) {
+                    continue;
+                }
+
+                for (var i = 0, l = aliases.length; i < l; i++) {
+                    brushes.push(aliases[i]);
+                }
+            }
+
+            if (brushes.indexOf(brush.toLowerCase()) == -1) {
+                $('.highlighter-output-broken').modal('', { width: 520 }).render();
```

Comment:
Let's make this a bit wider so, the header does not wrap for locales that generally use more characters per word, for example German :)